src/converter.js

```diff
@@ -154,8 +160,8 @@ converter._object = (name, node, mode) => {
     fields: fields
   }
 
-  if(node['decription']){
-    result.description = node['description']
+  if(node.decription){
```

Comment:
Misspelled `decription`. I suspect this is not covered (or is incorrectly covered) by the unit tests, otherwise it wouldn't have been missed. Can we fix and improve the test coverage to avoid a future regression?

Reply:
Fixed and test added